addons/account/views/account_payment_term_views.xml

```diff
@@ -79,7 +79,7 @@
                                     on
                                     <field name="example_date" class="fw-bold oe_inline"/>
                                 </div>
-                                <div colspan="2" style="background:#e0e0e0" class="pt4 pb4">
+                                <div colspan="2" style="background:rgba(0, 0, 0, 0.1)" class="pt4 pb4">
```

Comment:
```suggestion
                                <div colspan="2">
```

Reply:
Thanks for the review!! I agree on this, but I didn't know the context of why was on a different background 😄